.eslintrc.js

```diff
@@ -267,5 +267,21 @@ module.exports = {
         'import/no-extraneous-dependencies': 'off',
       },
     },
+    {
+      files: ['packages/core/src/tools/**/*.ts'],
+      rules: {
+        'no-restricted-imports': [
+          'error',
+          {
+            patterns: [
+              {
+                group: ['../**/boot/*', '../**/browser/*', '../**/domain/*', '../**/transport/*'],
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** [‏In the docs](https://eslint.org/docs/latest/rules/no-restricted-imports#options), it looks like you can allow some glob by starting with `!`. Maybe we can do something like `'../**', '!../tools/**'`

Reply:
Yes, I struggled for a while to try to make it work with some variations of: `'../**', '!../**/test'` but without success 😕
However, this strategy may not work if we have more directory levels inside `tools` so I settled with the current proposition.

Let me know if you have other ideas.

Reply:
No other idea, it sounds fine to me